com.unity.shadergraph/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/IsInfiniteNode.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections.Generic;
+using static UnityEditor.ShaderGraph.Registry.Types.GraphType;
+
+namespace com.unity.shadergraph.defs
+{
+
+    internal class IsInfiniteNode : IStandardNode
+    {
+        public static FunctionDescriptor FunctionDescriptor => new(
+            1,
+            "IsInfinite",
+            "Out = isinf(In);",
+            new ParameterDescriptor("In", TYPE.Float, Usage.In),
```

Comment:
This could be a `TYPE.Vector`

Reply:
It's not Vector in the old system.  If we made this a Vector, would we need to make the output a Bool4? We don't have a boolean data type that is of length Any - but I think that's what we would need if we needed to test each component of a variable length input vector.

Reply:
In the old node it's `        static string Unity_IsInfinite(
            [Slot(0, Binding.None)] Vector1 In,
            [Slot(1, Binding.None)] out Boolean Out)
        {
            return
@"
{
    Out = isinf(In);
}
";
        }` with a vector1 type.

Reply:
Oh yea-- it's fine as is. I'm not commenting to ask for a change, just laying out some thoughts-- eg. a node like this might make more sense to do: `Out = any(isinf(In));`.